src/components/Menu/index.tsx

```diff
@@ -103,29 +100,15 @@ const Menu: React.FC<{ chain?: number }> = () => {
     }
   }, [])
 
+  const buttonStyle = dynamicStyles.userBlockBtn({ showMenu })
+
   return (
     <>
-      <Box
-        sx={{
-          background: 'navbar',
-          position: 'fixed',
-          width: '100%',
-          height: '60px',
-          transform: `translateY(${showMenu ? 0 : '-100%'})`,
-          transition: 'transform linear 0.1s',
-          zIndex: 9,
-        }}
-      >
-        <Flex
-          sx={{ alignItems: 'center', height: '100%', justifyContent: 'space-between', px: ['25px', '25px', '20px'] }}
-        >
+      <Box sx={buttonStyle}>
```

Comment:
`buttonStyle` name does not match the component. It is not a button

Reply:
Changed name to menuStyle